src/packages/commit-lint/lib/lib.js

```diff
@@ -0,0 +1,70 @@
+const fs = require('fs');
+const { ArgumentParser } = require('@eigenspace/argument-parser');
+
+function exitWithSuccess(message) {
+    // eslint-disable-next-line no-console
+    console.info(message);
+    process.exit(0);
+}
+
+function exitWithError(message) {
+    // eslint-disable-next-line no-console
+    console.error(message);
+    process.exit(1);
+}
+
+function getCommitMessage() {
+    const parser = new ArgumentParser();
+    const args = parser.get(process.argv);
+
+    const message = args.get('message');
+
+    if (message === 'COMMIT_MESSAGE') {
+        return fs.readFileSync('.git/COMMIT_EDITMSG', 'utf8');
+    }
+
+    return message;
+}
+
+function getConfigContentFrom(path) {
+    let config = {};
+    try {
+        const content = fs.readFileSync(path, 'utf8');
+        config = JSON.parse(content);
+    } catch (err) {
+        exitWithError(`Failed to load ${path}`);
+        return;
+    }
+
+    return config;
+}
```

Comment:
```suggestion
function getConfigContentFrom(path) {
    try {
        const content = fs.readFileSync(path, 'utf8');
        return JSON.parse(content);
    } catch (err) {
        exitWithError(`Failed to load ${path}`);
    }
}
```

Also we can create method in @eigenspace/utils to read json like: `FsUtils.readJson(path)`. We use it very often.

Reply:
Planned to next release